boomer/boomer.go

```diff
@@ -127,6 +131,9 @@ func (b *Boomer) runWorker(n int) {
 		TLSHandshakeTimeout: time.Duration(b.Timeout) * time.Millisecond,
 		Proxy:               http.ProxyURL(b.ProxyAddr),
 	}
+	if b.H2 {
+		http2.ConfigureTransport(tr)
```

Comment:
if `TLSClientConfig` is set to nil (use default configuration), HTTP/2 support is enabled automatically. but if we set `InsecureSkipVerify: true` in tls.Config, It seems that we can only use `http2.ConfigureTransport` to add HTTP/2 support.